molo/core/views.py

```diff
@@ -136,8 +136,10 @@ def add_translation(request, page_id, locale):
     language_relation = translation.languages.first()
     language_relation.language = language
     language_relation.save()
+    page.translated_pages.add(translation)
```

Comment:
So the templates should also be updated to relfect that the translations are done in the view